nextflow.config

```diff
@@ -20,51 +16,76 @@ params {
 
     // For already fetched data
     samplesheet                = null
-    /*
-    * Assembler options, by default the pipeline will pick
-    * - metaspades for pair-end
-    * - megahit for single-end
-    *
-    * Setting --assembler will force the assembler
-    *
-    * - spades: Use for assembling single end reads
-    *
-    * - metaspades: Use for assembling paired end reads
-    *   with moderate memory and runtime requirements
-    *
-    * - megahit: Use when memory or runtime requirements
-    *   for metaspades are prohibitively high, such as:
-    *    - Memory >1TB
-    *    - Runtime >3-4 days
-    */
-    assembler                        = null
 
     // The pipeline will use the metadata from ENA (obtained by the fetch_tool)
     // As the metadata can be incorrect, we provide the following parameters to
     // "force" them
     single_end                       = null
     library_layout                   = null
     library_strategy                 = null
+    platform                         = null
+
+    // QC FILTERING
 
-    // Reads QC filtering options
-    filter_ratio_threshold           = 0.9
-    low_reads_count_threshold        = 1000
+    // Short reads QC filtering options
+    short_reads_filter_ratio_threshold    = 0.1
+    short_reads_low_reads_count_threshold = 1000
 
-    // Reference genome
-    reference_genome                 = null
+    // Long reads options
+    long_reads_min_read_length        = 200
 
+    // Short reads reference databases (name to be selected from list)
+    bwamem2_reference_genomes_folder = ""
+    blast_reference_genomes_folder   = ""
+
+    // Long reads reference genome
+    reference_genome            = null
```

Comment:
Is this only used for LR then? Should it be `long_reads_reference_genome` for consistency with other param naming?